README.md

```diff
@@ -87,10 +87,20 @@ npm start
   ]
 ]
 ```
+
+* `CONNECTOR_NOTIFICATION_VERIFY` (default: `'true'`) The connector verifies the signatures on notifications.
```

Comment:
I don't buy this argument. They need to explicitly set this to false. They could also make a number of other bad decisions, like not running HTTPS, or incorrectly configuring the public keys.